packages/webapp/src/inductions/components/induction-lists/action-button.tsx

```diff
@@ -0,0 +1,25 @@
+import NextLink from "next/link";
+
+interface Props {
+    href: string;
+    className?: string;
+    lightText?: boolean;
+    children: React.ReactNode;
+}
+
+export const InductionActionButton = ({
```

Comment:
out of curiosity: any reason why we can't make this button a generic one inside _app/ui? (or even merge/mix with the current button?) Just wondering because I think we don't want to have different non-standard types of the buttons across the ui.

Reply:
I wanted to keep this PR scoped as minimally as possible and tread lightly. But we're on the same wavelength...I'd like for this simpler button to potentially replace the other one across the UI. And I'm planning to experiment with that in a future PR. I'll keep this here for now...but you'll likely see this change happen very soon.